parser_library/src/config/proc_grps.h

```diff
@@ -39,6 +39,8 @@ struct assembler_options
     std::string profile;
     std::string optable;
     std::string system_id;
+    bool goff = false;
+    bool xobject = false;
```

Comment:
This is the only part I am not sure about... The keywords are synonyms, should we therefore keep two state variables?